src/sources/grib.jl

```diff
@@ -0,0 +1,223 @@
+const CFG = CfGRIB
+
+const GRIB_DIMMAP = Dict(
```

Comment:
I made it more specific in the last commit. I think I can't take the same as for netcdf since the relevant `time` dimension is usually `valid_time` in GRIB (as far as I know). Or maybe I should just transform the dimension name from `valid_time` to `time` in GRIBDatasets.jl, and then we could use the same dimension mapping

Reply:
Its ok to keep the standard names.